examples/README.md

```diff
@@ -0,0 +1,85 @@
+# Examples
+
+*Note*: Unless specified otherwise, all commands should be executed from within this folder. It is assumed that job-executor-service is arleady installed and running (in remote execution plane).
+
+## Hello World
```

Comment:
I think it would be best if the examples were self contained and these instructions were included in a README.md in the example subfolder (personal preference, I know)

Reply:
That's exactly where the README is :) Not sure if we are talking about different things